scripts/fund-dev-accounts/main.rs

```diff
@@ -0,0 +1,153 @@
+/// Therefore, after network launch there needs to be a reserve transfer from the relay chain
+/// to the dev accounts.
+///
+/// This script performs these reserve transfers to fund the dev accounts from the relay chain.
+///
+use subxt::{OnlineClient, PolkadotConfig};
+use subxt_signer::sr25519::{dev, Keypair};
+
+#[cfg(not(feature = "paseo"))]
+mod relay {
+	use super::*;
+	#[subxt::subxt(runtime_metadata_path = "../metadata/rococo.scale")]
+	pub(crate) mod runtime {}
+	pub(crate) type RuntimeCall = runtime::runtime_types::rococo_runtime::RuntimeCall;
+	pub(crate) const UNIT: u128 = 1_000_000_000_000;
+
+	pub(crate) use runtime::runtime_types::{
+		staging_xcm::v4::{
+			asset::Fungibility::Fungible,
+			asset::{Asset, AssetId, Assets},
+			junction::Junction,
+			junctions::Junctions,
+			junctions::Junctions::X1,
+			location::Location,
+		},
+		xcm::{v3::WeightLimit, VersionedAssets, VersionedLocation},
+	};
+
+	// generate XCM message to reserve transfer funds to a designated account on
+	// Pop Parachain
+	pub(crate) fn gen_account_fund_message_call(account: Keypair) -> RuntimeCall {
+		let pop_location = VersionedLocation::V4(Location {
+			parents: 0,
+			interior: X1([Junction::Parachain(9090)]),
+		});
+		let pop_beneficiary = VersionedLocation::V4(Location {
+			parents: 0,
+			interior: X1([Junction::AccountId32 { network: None, id: account.public_key().0 }]),
+		});
+		let amount = Fungible(UNIT * 1_000_000);
+		let assets = VersionedAssets::V4(Assets {
+			0: vec![Asset {
+				id: AssetId { 0: Location { parents: 0, interior: Junctions::Here } },
+				fun: amount,
+			}],
+		});
+
+		RuntimeCall::XcmPallet(
+			crate::relay::runtime::xcm_pallet::Call::limited_reserve_transfer_assets {
+				dest: Box::new(pop_location),
+				beneficiary: Box::new(pop_beneficiary),
+				assets: Box::new(assets),
+				fee_asset_item: 0,
+				weight_limit: WeightLimit::Unlimited,
+			},
+		)
+	}
+}
+
+#[cfg(feature = "paseo")]
+mod relay {
+	use super::*;
+
+	#[subxt::subxt(runtime_metadata_path = "../metadata/paseo.scale")]
+	pub(crate) mod runtime {}
+	pub(crate) type RuntimeCall = runtime::runtime_types::paseo_runtime::RuntimeCall;
+	pub(crate) const UNIT: u128 = 10_000_000_000;
+
+	use runtime::runtime_types::{
+		staging_xcm::v3::multilocation::MultiLocation as Location,
+		xcm::{
+			v3::{
+				junction::Junction,
+				junctions::Junctions,
+				junctions::Junctions::X1,
+				multiasset::{
+					AssetId::Concrete, Fungibility::Fungible, MultiAsset as Asset,
+					MultiAssets as Assets,
+				},
+				WeightLimit,
+			},
+			VersionedMultiAssets as VersionedAssets, VersionedMultiLocation as VersionedLocation,
+		},
+	};
+
+	// generate XCM message to reserve transfer funds to a designated account on
+	// Pop Parachain
+	pub(crate) fn gen_account_fund_message_call(account: Keypair) -> RuntimeCall {
+		let pop_location =
+			VersionedLocation::V3(Location { parents: 0, interior: X1(Junction::Parachain(9090)) });
```

Comment:
As above, `9090` should be changed in line with what is set in the spec.

(unfortunately)